src/NextFederationPlugin.js

```diff
@@ -360,45 +358,9 @@ function generateRemoteTemplate(url, global) {
         __webpack_error__.type = errorType;
         __webpack_error__.request = realSrc;
         reject(__webpack_error__);
-      },
-      ${JSON.stringify(global)},
+      }, ${JSON.stringify(global)},
     );
-  }).then(function () {
-    const proxy = {
-      get: ${global}.get,
-      init: (args) => {
```

Comment:
We muat keep the proxy otherwise shared modules are loaded twice since webpack has no async boundaries in next. 

If you start demo app and click on test hook or shop links. It can fail without the proxy. If remotes are under other entrypoints webpack cannot initialize all containers at the right time. 

We need to mark the host modules as loaded via proxy too to prevent loading things that already shared by host